src/style/_default.scss

```diff
@@ -25,8 +25,21 @@ SPDX-License-Identifier: Apache-2.0
 body {
   height: 100vh;
   width: 100vw;
+  background-color: #eee;
```

Comment:
I think this could be `var(--light)` to re-use the value from the Material theme.

Reply:
Resolved with #5bc55af8